include/edge_utils_win32.h

```diff
@@ -21,9 +21,12 @@
 
 #ifdef WIN32
 
+#define WIN32_LEAN_AND_MEAN
```

Comment:
What is this define? Why is necessary?

Reply:
ref:  https://issues.apache.org/jira/browse/THRIFT-2150
When both Windows.h and Winsock2.h are included, it is easy to cause compilation conflicts.